modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -50,7 +64,7 @@ $ oc patch Network.config.openshift.io cluster \
 +
```

Comment:
line 63 shall also be updated. Since we will watch the pods of multus, not openshift-sdn.